R/metadata-extraction_imagery_dataset.R

```diff
@@ -80,7 +86,7 @@ extract_camera_pitch_summary <- function(exif) {
   # just in case the pitch is reported as negative for backward (even thouth it apparently never
   # is), so that for smart-oblique missions, the low quantile represents nadir and the high quantile
   # represents the oblique mission pitch
```

Comment:
Could you move this comment about adjusting for nadir to the image-level extraction where the +90 happens? I realize I didn't comment it there, but no this comment here is irrelevant.